cli/src/main/scala/aqua/files/AquaFilesIO.scala

```diff
@@ -107,7 +107,8 @@ class AquaFilesIO[F[_]: Files: Concurrent] extends AquaIO[F] {
         } else {
           Files[F].isDirectory(folder).flatMap { isDir =>
             if (isDir) {
-              Files[F].list(folder).compile.toList.map(Right(_))
+              Files[F].list(folder).evalFilter(p => Files[F].isDirectory(p).map(isDir => isDir || p.extName == ".aqua"))
```

Comment:
Can be optimized by checking for extName first, and only if it's not `.aqua` access file system for `isDirectory` check